### PR TITLE
Slot return fix

### PR DESCRIPTION
**Pull request by AnujaLK**
Jira - https://wso2.org/jira/browse/MB-1156

### CODE REVIEW

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/SlotManagerClusterMode.java [hunk 1]
```diff
@@ -152,16 +146,13 @@ public Slot getSlot(String queueName, String nodeId) throws AndesException{
...
                 slotToBeAssigned = getFreshSlot(queueName, nodeId);
-                isFreshSlot = true;
             }
         }
 
         if (null != slotToBeAssigned) {
-            if (!isFreshSlot) {
-                updateSlotAssignmentMap(queueName, slotToBeAssigned, nodeId);
-            }
+	        updateSlotAssignmentMap(queueName, slotToBeAssigned, nodeId);
```
Comment (a5anka) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/SlotManagerClusterMode.java]: Please check if the indentation is correct.

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/SlotManagerClusterMode.java [hunk 2]
```diff
@@ -152,16 +146,13 @@ public Slot getSlot(String queueName, String nodeId) throws AndesException{
...
             if (log.isDebugEnabled()) {
-                log.debug("Assigning slot for node : " + nodeId + " ||| " + slotToBeAssigned);
+                log.debug("Assigning slot for node : " + nodeId + " | " + slotToBeAssigned);
```
Comment (a5anka) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/SlotManagerClusterMode.java]: Please check if the indentation is correct.

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/SlotManagerClusterMode.java (hunk 1 above)
Comment (AnujaLK) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/SlotManagerClusterMode.java]: Fixed with PR https://github.com/wso2/andes/pull/276/

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/SlotManagerClusterMode.java (hunk 2 above)
Comment (AnujaLK) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/SlotManagerClusterMode.java]: Fixed with PR https://github.com/wso2/andes/pull/276/

**Pull request merged**